com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -987,18 +987,6 @@ protected void SetupRenderPasses(in RenderingData renderingData)
 
                 rendererFeatures[i].SetupRenderPasses(this, in renderingData);
             }
-
-            // Remove any null render pass that might have been added by user by mistake
-            int count = activeRenderPassQueue.Count;
-            for (int i = count - 1; i >= 0; i--)
-            {
-                if (activeRenderPassQueue[i] == null)
-                    activeRenderPassQueue.RemoveAt(i);
-            }
```

Comment:
Why do we remove this?

Reply:
this was the copy paste error in the RTHandles PR which disabled by mistake all the store actions optimizations (the same code is already in AddRenderPasses)